Base/QTGUI/qSlicerSettingsGeneralPanel.h

```diff
@@ -52,7 +52,9 @@ public slots:
   void openSlicerRCFile();
 
 protected slots:
+#ifdef Slicer_BUILD_APPLICATIONUPDATE_SUPPORT
```

Comment:
Could you check that moc correctly regenerate the `moc_qSlicerSettingsGeneralPanel.cpp` file by deleting it ? 

Explicitly including `vtkSlicerConfigure.h` may also be sensible.

In the past, we had to explicitly pass macro to moc.

Reply:
Per @lassoan suggestion, if this is effectively an issue, I think it may be easier to only exclude the code in the implementation.

Reply:
I've pushed the update to exclude the code in `updateAutoUpdateApplicationFromManager` instead